packages/cyberstorm/src/components/Footer/Footer.tsx

```diff
@@ -23,7 +16,7 @@ import { Tooltip } from "../Tooltip/Tooltip";
 import { Icon } from "../Icon/Icon";
 import { classnames } from "../../utils/utils";
 
-const AD_IMAGE_SRC = "/images/tsmm_screenshot.png";
+const AD_IMAGE_SRC = "/cyberstorm/images/tsmm_screenshot.png";
```

Comment:
Not important here but if we use more static images before the old website is completely replaced, we might want to have the `/cyberstorm/images` path in a global static variable so we can edit the path in one place when the cyberstorm-prefix is dropped.

Reply:
Hmm, I'm not sure right now how to do separation between component library and nextjs right now, so I'll add it to be figured out later. But yes this is a thing that needs to be sorted out.